phoenix-core/src/it/java/org/apache/phoenix/end2end/DerivedTableIT.java

```diff
@@ -313,21 +290,25 @@ public void testDerivedTableWithGroupBy() throws Exception {
             query = "SELECT DISTINCT COLLECTDISTINCT(t.b) FROM (SELECT b_string b, a_string a FROM aTable GROUP BY a_string, b_string) AS t GROUP BY t.a";
```

Comment:
@JamesRTaylor Please ignore my previous (deleted) comment. I just realized that the server group-by has replaced tree map with hash map, which means an optimization I did for client group-by based on the assumption that the input is a sorted group-by is now invalid. I will fix this ASAP. And after the fix, the test case should fine, we should keep it as it is.